lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -759,7 +764,7 @@ export async function ensureCommentRemoval(
 // Pull Request
 
 const escapeHash = (input: string): string =>
-  input ? input.replace(regEx(/#/g), '%23') : input;
+  input?.replace(regEx(/#/g), '%23');
```

Comment:
Previously, if `input` was an empty string then this would return an empty string. I think this case it will return `undefined`?

Reply:
it's now the same. `?.` operator returns only when `null` or `undefined` is passed as `input`. otherwise it will try to replace in empty string, which is of cause an empty string again. so no real change here, just a syntactic shortcut.

i know the type declaration is wrong, as it should accept and return `null` and `undefined`., otherwise the null check wouldn't necessary.

will add a Todo comment to fix types. will cause to many changes for now.